README.md

```diff
@@ -47,9 +48,12 @@ or
 
     sudo port install libtool mariadb-server rabbitmq-server
 
-(On OS X, using port, you will have to add `CGO_CFLAGS="-I/opt/local/include" CGO_LDFLAGS="-L/opt/local/lib"` to your environment or `go` invocations.)
+Note: 
 
+(On OS X, using port, you will have to add `CGO_CFLAGS="-I/opt/local/include" CGO_LDFLAGS="-L/opt/local/lib"` to your environment or `go` invocations.)
+   
     > go get bitbucket.org/liamstask/goose/cmd/goose
+    > go get github.com/jsha/listenbuddy
```

Comment:
@bifurcation @jsha this is required by startserver.py, but based on the comment, it seems like it's not necessary for production operations.